packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -144,6 +145,11 @@ export function getGCVersion(metadata?: IContainerRuntimeMetadata): GCVersion {
     return metadata.gcFeature ?? 0;
 }
 
+// eslint-disable-next-line prefer-arrow/prefer-arrow-functions
+export function getRuntimeVersion(metadata?: IContainerRuntimeMetadata): string {
```

Comment:
unless there is a scenario where this API is needed, I'd not add it, as we can't easily remove it later (as we do not know who would start using it). At the moment, the only place for this info I see is telemetry, i.e. no need to expose outside of the package.